packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -249,7 +249,7 @@ class ThemeData extends Diagnosticable {
       splashColor: splashColor,
       materialTapTargetSize: materialTapTargetSize,
     );
-    disabledColor ??= isDark ? Colors.white30 : Colors.black38;
+    disabledColor ??= isDark ? Colors.white50 : Colors.black38;
```

Comment:
This change is likely to have an impact on many apps and tests. Even though it's a tiny change, it would be good to move it to a separate PR.

Reply:
Tracking here: https://github.com/flutter/flutter/issues/33149